src/main/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagApp.scala

```diff
@@ -33,12 +33,18 @@ class EasyValidateDansBagApp(configuration: Configuration) extends DebugEnhanced
   private val schemaFactory = SchemaFactory.newInstance("http://www.w3.org/2001/XMLSchema")
   logger.info("XML Schema factory created.")
 
-  private def createValidator(schemaUrl: URL): XmlValidator = Try {
-    logger.info(s"Creating validator for $schemaUrl ...")
-    val ddmSchema = schemaFactory.newSchema(schemaUrl)
-    val v = new XmlValidator(ddmSchema)
-    logger.info("Validator created.")
-    v
+  private def createValidator(schemaUrl: URL): XmlValidator = {
+    logger.info(s"Creating validator for $schemaUrl with agent ${ System.getProperty("http.agent") } ...")
+    for {
+      ddmSchema <- Try(schemaFactory.newSchema(schemaUrl))
+      xmlValidator <- Try(new XmlValidator(ddmSchema))
+      _ = logger.info(s"Validator created with $schemaUrl")
+    } yield xmlValidator
+  }.doIfFailure {
+    case e if e.getMessage.contains("Cannot resolve") =>
+      logger.error(s"Probably an offline third party schema for $schemaUrl : ${ e.getMessage }", e)
```

Comment:
"Probably an offline third party schema" seems like jumping to conclusions too fast. What if it turns out to be something else? Then this formulation of the error message will only add unnecessary confusion.

I think this "case" is actually not helpful. If the creation fails we will have to look at the logs anyway.

Reply:
This is an old habit, at least used a lot for `assume`s to ignore unit tests. Without recent experience the error message is a riddle, for example: 

    SAXParseException; lineNumber: 158; columnNumber: 71; src-resolve: Cannot resolve the name 'dc:description' to a(n) 'element declaration' component.

without recent experience you may look in vain for changed/invalid schema's, while in practice the added hint is the cause. I could reduce `probably` to `possibly` and check for the exception class. Having said that: we now know it is not just about an offline schema but could also be an agent refused by a DC schema.